services/search/src/search/routes/filter.py

```diff
@@ -123,15 +123,22 @@ async def filter_endpoint(request: Request) -> Response:
                             error_code=duckdb_index_cache_entry["error_code"],
                             revision=revision,
                         )
+
+                    # check if the index is on the full dataset or if it's partial
```

Comment:
we have the info in `duckdb_index_cache_entry`, no?

Reply:
not always - I added the `partial` field for this PR and didn't increase the job version

Reply:
yes, but if I understand correctly, if the duckdb has the `partial` keyword in its name, it also has the field in `duckdb_index_cache_entry`

Reply:
If it doesn't have the `partial` keyword in its name, the split directory name can still have it have it

Reply:
ok